testsuite/domain/src/test/resources/host-configs/host-slave-auto-start.xml

```diff
@@ -78,7 +78,7 @@
 
     <domain-controller>
         <!-- Remote domain controller configuration with a host and port -->
-        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm">
+        <remote host="${jboss.test.host.master.address}" port="9999" security-realm="ManagementRealm" ignore-unused-configuration="false">
```

Comment:
I'm concerned if this has to be changed. It's only used in ServerAutoStartTestCase and I don't see anything in there that should fail if the slave is ignoring. If it fails, that may mean there's a bug.